Assert less specifically against the propshaft route

Earlopain opened the pull request:
Propshaft 1.1.0 got released, which (obviously) changed the version and moved some routes around.

```
Failure:
Rails::Command::RoutesTest#test_rails_routes_with_expanded_option [test/commands/routes_test.rb:249]:
--- expected
+++ actual
@@ -3,7 +3,7 @@
 Verb              |
 URI               | /assets
 Controller#Action | Propshaft::Server
-Source Location   | propshaft (1.0.0) lib/propshaft/railtie.rb:43
+Source Location   | propshaft (1.1.0) lib/propshaft/railtie.rb:47
 --[ Route 2 ]--------------
 Prefix            | cart
 Verb              | GET
```

I don't think it's easily available in that context where a route is actually defined, so just skip checking that one specific line.

This is visible in `rack-head` test that probably remove the lockfile (it's allowed to fail though). https://buildkite.com/rails/rails/builds/112060#0192480d-dbb0-4ead-9cf5-5451af14cae6

zzak commented on railties/test/commands/routes_test.rb:
```diff
@@ -239,20 +239,25 @@ class Rails::Command::RoutesTest < ActiveSupport::TestCase
 
-    # rubocop:disable Layout/TrailingWhitespace
-    assert_equal <<~MESSAGE, output
+    propshaft_route = lines[0..6].join
+    remaining_routes = lines[6..].join
```
I think this could create some weird edge case when the output changes again, or propshaft adds/removes routes.

Can you think of another way to filter the routes?

Thanks for working on this!

Earlopain replied:
Took a peek at how propshaft does it and the presence of routes relies on `config.assets.server`. So setting that to false skips drawing them which makes this change much less brittle

Earlopain committed:
Don't assert against propshaft routes

Propshaft 1.1.0 got released, which (obviously) changed the version
and moved some routes around.

```
Failure:
Rails::Command::RoutesTest#test_rails_routes_with_expanded_option [test/commands/routes_test.rb:249]:
--- expected
+++ actual
@@ -3,7 +3,7 @@
 Verb              |
 URI               | /assets
 Controller#Action | Propshaft::Server
-Source Location   | propshaft (1.0.0) lib/propshaft/railtie.rb:43
+Source Location   | propshaft (1.1.0) lib/propshaft/railtie.rb:47
 --[ Route 2 ]--------------
 Prefix            | cart
 Verb              | GET
```

Earlopain commented:
https://github.com/rails/rails/pull/53162
